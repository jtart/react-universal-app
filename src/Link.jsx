import React from 'react';
import { Link } from 'react-router-dom';

export default ({ to, children }) => <Link to={to}>{children}</Link>;
