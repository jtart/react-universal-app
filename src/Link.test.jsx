import React from 'React';
import Link from './Link';

describe('Link', () => {
  it('should render a React Router link', () => {
    const wrapper = shallow(<Link to="/test">Test!</Link>);
    expect(wrapper).toMatchSnapshot();
  });
});
