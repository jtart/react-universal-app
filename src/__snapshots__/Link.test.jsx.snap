// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link should render a React Router link 1`] = `
<Link
  replace={false}
  to="/test"
>
  Test!
</Link>
`;
